src/EditorFeatures/Core/Extensibility/Navigation/NavigableItemFactory.DeclaredSymbolNavigableItem.cs

```diff
@@ -53,6 +51,22 @@ public DeclaredSymbolNavigableItem(Document document, DeclaredSymbolInfo declare
                     }
                 });
             }
+
+            private ISymbol FindSymbol()
+            {
+                // Here, we will use partial semantics. We are going to use this symbol to get a glyph, display string,
+                // and potentially documentation comments. The first two should work fine even if we don't have full
+                // references, and the latter will probably be fine. (It wouldn't be if you have a partial type
+                // and we didn't get all the trees parsed yet to know that. In other words, an edge case we don't care about.)
```

Comment:
The fortunate thing is that error types look the same as normal types in display strings :)   i.e. if we have a method like ```void Foo(Blah<X>)```, then even if we can't bind Blah or X, we'll still present things as ```void Foo(Blah<X>)```